docker-compose.hobby.yml

```diff
@@ -31,7 +31,7 @@ services:
             - '9440:9440'
             - '9009:9009'
         volumes:
-            - ./posthog/ee/idl:/idl
+            - ./posthog/posthog/idl:/idl
```

Comment:
a review from infra here just to sanity check this is fine! (cc: @tiina303 , @yakkomajuri )

Reply:
Well I'm about to [nuke protobuf](#9207) but currently nobody really knows much about the setup besides @fuziontech.

From my understanding you might also need to update https://github.com/PostHog/posthog/blob/master/docker/clickhouse/docker-entrypoint-initdb.d/init-db.sh?

Best to check all this with @fuziontech